main/options.c

```diff
@@ -481,53 +481,58 @@ static const char* const License2 =
 /*  Contains a set of strings describing the set of "features" compiled into
  *  the code.
  */
-static const char *const Features [] = {
+static struct Feature {
+	const char *name;
+	const char *description;
+} Features [] = {
 #ifdef WIN32
-	"win32",
+	{"win32", "TO BE WRITTEN"},
 #endif
-	"wildcards",		/* Always available on universal ctags */
-	"regex",		/* Always available on universal ctags */
+	/* Following two features are always available on universal ctags */
+	{"wildcards", "can use glob matching"},
+	{"regex", "can use regular expression based pattern matching"},
+
 #ifndef EXTERNAL_SORT
-	"internal-sort",
+	{"internal-sort", "uses internal sort routine instead of invoking sort command"},
 #endif
 #ifdef CUSTOM_CONFIGURATION_FILE
-	"custom-conf",
+	{"custom-conf", "read \"" CUSTOM_CONFIGURATION_FILE "\" as config file"},
 #endif
 #if defined (WIN32) && defined (UNIX_PATH_SEPARATOR)
-	"unix-path-separator",
+	{"unix-path-separator", "can use '/' as file name separator"},
 #endif
 #ifdef HAVE_ICONV
-	"multibyte",
+	{"multibyte", "TO BE WRITTEN"},
 #endif
 #ifdef DEBUG
-	"debug",
+	{"debug", "TO BE WRITTEN"},
 #endif
 #ifdef HAVE_SCANDIR
-	"option-directory",
+	{"option-directory", "TO BE WRITTEN"},
 #endif
 #ifdef HAVE_LIBXML
-	"xpath",
+	{"xpath", "linked with library for parsing xml input"},
```

Comment:
...Good point.

Originally, I wanted to add `--xpath-<LANG>=...` option like `--regex-<LANG>=` option; I think it is useful feature that people can create a parser with specifying xpaths.
However, I have not implemented it yet because I cannot find how the option parameters should be.
What I have done is some libxml2 base parser. In the future, when I implement `--xpath-<LANG>` option, the feature "xpath" is the signed that a ctags binary can accept `--xpath` option.